ingestion/functions/test_clean_old_ingestion_source_files.py

```diff
@@ -0,0 +1,77 @@
+import boto3
+import os
+import pytest
+import unittest
+from datetime import datetime, timedelta
+from botocore.exceptions import ClientError
+
+import clean_old_ingestion_source_files
+import aws_access.globaldothealth_configuration as gdoth
+
+localstack = 'http://localhost:4566'
+rate_event_id = '5f7796ece78c6866f6f676e0'
+file_content = "blah blah blah" # this is unimportant
+file_time = '1130' # also unimportant
+file_name = 'slartibartfast.json' # "my name is not important"
+
+def s3_object_key_helper(source_id, date, file_name):
+    return f"{source_id}/{date.year}/{date.month}/{date.day}/{file_time}/{file_name}"
+
+
```

Comment:
Should all methods only run in the dockerized environment?

Reply:
As this is called by the test and the test only runs in the docker environment, I think that's already true? Nothing is going to run this function otherwise.